components/FoodCard.js

```diff
@@ -0,0 +1,17 @@
+import Image from "next/image";
+import Link from "next/link";
+import React from "react";
+import styled from "styled-components";
+import salmon from "../assets/images/salmon.jpg";
```

Comment:
You do not need this import here. The image source is passed as a prop to this component (inside `data`).
```suggestion

```